src/EA.Weee.RequestHandlers/AatfReturn/ObligatedSentOn/GetWeeeSentOnHandler.cs

```diff
@@ -42,14 +42,29 @@ public async Task<List<WeeeSentOnData>> HandleAsync(GetWeeeSentOn message)
 
```

Comment:
Apparently the tests for this class don't cover most of the between 38 and 65.  Perhaps in the tests you need to have items in the sentOnList in HandleAsync_GivenGetSentOnAatfSiteRequest_AddressDataShouldBeMapped()